doc/plugin_tutorial/tuto1/src/inspector.ml

```diff
@@ -0,0 +1,8 @@
+open Pp
+
+(*
+ * Inspect an input and print a feedback message explaining what it is
+ *)
+let print_input (a : 'a) (printer : 'a -> Pp.t) (type_str : string) : unit =
```

Comment:
Not sure there's much point in the type annotations here.

Reply:
I usually include type annotations as much as possible because they are self-documenting